packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -0,0 +1,311 @@
+
+> **Note**: Currently, only the Okta API can be used to enable OAuth 2.0-based provisioning.
+---
+
+**Learning outcomes**
+
+* Set up a multi-tenant hub and spoke Okta org configuration for provisioning by using the OAuth 2.0 client credential grant flow
+* Configure hub Okta org with service apps for each spoke Okta org.
+* Configure Org2Org app on each spoke Okta org.
+* Configure and activate Org2Org provisioning on spoke Okta orgs.
+* Rotate keys for the OAuth 2.0 connection.
+
+**What you need**
+
+* Multiple Okta orgs for your multi-tenant solution
+
+---
+
+## About using OAuth 2.0 for secure API connections in multi-tenant solutions
+
+To secure API connections between orgs for a [multi-tenant solution](/docs/concepts/multi-tenancy/) that uses the hub and spoke model, Okta provides the OAuth 2.0 client credential flow using the Okta Org2Org integration as an OAuth 2.0 client. This enables Okta admins to push and match both users and groups from one Okta org to another. Previously, this integration only supported token-based access to the Okta API. Now the Org2Org integration can be configured to access the Okta API as an OAuth 2.0 client. This increases security by limiting the scope of access and providing a better mechanism to rotate credentials.
+
+![Displays the service apps required in the hub org and the Org2Org apps required in the spoke org.](/img/multi-tenancy/multi-org-provisioning.png "OAuth 2.0 connection configuration for hub and spoke org model")
+
+In this OAuth 2.0 flow, the hub org acts as the resource server as well as the [authorization server](/docs/concepts/auth-servers/).  Each spoke org is represented by a [service app in the hub org for the OAuth 2.0 client credentials flow](/docs/guides/implement-oauth-for-okta-serviceapp/main/). At the spoke org level, you need to create an Org2Org app to represent the OAuth 2.0 client. The Okta Org2Org app automatically generates a JSON Web Key Set (JWKS)  used by the client to authenticate with the authorization server. For each Org2Org app, you need to extract the JWKS public key for each corresponding service app in the hub org. To limit access to the OAuth 2.0 clients, you need to grant the hub org service app with allowed [OAuth 2.0 scopes](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints).
+
+After the Org2Org OAuth 2.0 connection is configured, you can start to test your connection and push user data from the spoke orgs to the hub org. You can also create a [rotate keys](#key-rotation) schedule to update your credentials on a regular basis.
+
+> **Note**: For a hub and spoke org model provisioning using API tokens instead of OAuth 2.0, see [Integrate Okta Org2Org with Okta ](https://help.okta.com/okta_help.htm?id=ext-org2org-intg).
```

Comment:
Maybe:

"For provisioning in a hub and spoke org model that uses API tokens instead of OAuth 2.0...

Reply:
Sounds good. Thanks!  👍